src/contacts/EditContact/EditContact.js

```diff
@@ -143,16 +148,27 @@ class EditContact extends Component {
                   <Col xs={3}>
                     <Field
                       component={Select}
+                      dataOptions={[{ label: '', value: '' }]}
                       fullWidth
                       label={<FormattedMessage id="ui-organizations.contactPeople.status" />}
                       name="inactive"
-                    />
+                    >
+                      {Object.keys(CONTACT_STATUSES).map((key) => (
+                        <FormattedMessage
+                          id={`ui-organizations.contactPeople.status.${key}`}
+                          key={key}
+                        >
+                          {(message) => <option value={CONTACT_STATUSES[key]}>{message}</option>}
+                        </FormattedMessage>
+                      ))}
+                    </Field>
                   </Col>
                 </Row>
                 <Row>
                   <Col xs={3}>
                     <Field
                       component={Select}
+                      dataOptions={this.getDropdownList(languageList)}
```

Comment:
you don't need `this` for `getDropdownList`

Reply:
Updated